src/Navbar.jsx

```diff
@@ -82,4 +91,5 @@ const Navbar = ({ sessionToken, deleteSessionToken }) => {
   );
 };
 
-export default Navbar;
+const CookieNavbar = withCookies(Navbar);
```

Comment:
Can be written as `export default withCookies(Navbar);` don't need to assign it to a variable